docs/literate/explanations/window_length.jl

```diff
@@ -0,0 +1,128 @@
+		sfreq = sfreq,
+		n1 = (n170(;sfreq=sfreq), @formula(0~1),[5],Dict()),
+		p3 = (p300(;sfreq=sfreq), @formula(0~1),[5],Dict()),
+		n_repeats=1,noiselevel=0, return_epoched=true);
+time_gt = range(0,length = length(dat_gt[:,1]),step=1/sfreq)
+df_gt = DataFrame(
+    basisname = reduce(vcat,fill.(unique(res.basisname), length(dat_gt[:,1]))),
+    channel = repeat([1], length(dat_gt[:,1])*length(unique(res.basisname))),
+    coefname = reduce(vcat,fill("GroundTruth", length(dat_gt[:,1])*length(unique(res.basisname)))),
+    estimate = repeat(dat_gt[:,1], length(unique(res.basisname))),
+    group = reduce(vcat,fill(nothing, length(dat_gt[:,1])*length(unique(res.basisname)))),
+    stderror = reduce(vcat,fill(nothing, length(dat_gt[:,1])*length(unique(res.basisname)))),
+    time = repeat(time_gt, length(unique(res.basisname))),
+        );
+
+df_gt.tWin .= basisToWin(df_gt.basisname);
+
+# And append ground truth to our results df	
+res_gt = vcat(res, df_gt);
+
+
+# # Plot results
+
+# Plot figure
+f = Figure();
+	
+# Choose which data to plot
+h_t = data(res) * mapping(:time,:estimate,color=:tWin,group=(:tWin,:coefname)=>(x,y)->string(x)*y);
+
+# We use the following to plot some length indicator lines
+untWin = unique(res_gt.tWin)
+segDF = 	DataFrame(
+    :x=>hcat(repeat([-0.1],length(untWin)),untWin)'[:],
+    :y=>repeat(reverse(1:length(untWin)),inner=2))
+segDF.tWin .= 0.
+segDF.tWin[1:2:end] .= segDF.x[2:2:end]
+segDF.tWin[2:2:end] .= segDF.x[2:2:end]
+segDF.y = segDF.y .* 0.2 .+ 6;
+
+# Layer for indicator lines
+h_l = data(@subset(segDF,:tWin .!=3.0))* mapping(:x,:y,color=:tWin,group=:tWin=>x->string.(x));
+
+# Ground truth Layer
+h_gt = data(df_gt) * mapping(:time,:estimate,group=(:tWin,:coefname)=>(x,y)->string(x)*y)*visual(Lines,linewidth=5,color=Colors.Gray(0.6));
+
+# Add all visuals together and draw
+h1 = h_gt + visual(Lines,colormap=get(ColorSchemes.Blues,0.3:0.01:1.2)) * (h_l + h_t) |> x->draw!(f[1,1],x,axis=(;xlabel="time [s]",ylabel="estimate [a.u.]"));
+
+# Add zero grid lines
+h1 = hlines!(current_axis(),[0],color=Colors.Gray(0.8));
+h2 = vlines!(current_axis(),[0],color=Colors.Gray(0.8));
+translate!(h1, 0, 0, -1);
+translate!(h2, 0, 0, -1);
+
+# Plot figure
+current_figure()
+
+# # Conclusion & further information
+#
+# For analyzing real-world EEG data we recommend that researchers should — a priori — make an educated guess about the length of the underlying EEG activity and select this as their EW. This also suggests to use event windows with different sizes between events (as is possible with Unfold).
+# Further, as can be seen above, when choosing longer time-windows the overfit is only of moderate size, thus we additionally recommend to generally err on the longer side, to not miss any important activity.
+#
+# For a more in depth explanation on this, you can read our 2023 CCN paper: [Skukies & Ehinger, 2023](https://www.biorxiv.org/content/10.1101/2023.06.05.543689v1)
```

Comment:
I would put this reference to the very top somewhere also with a !!! important distinction
```
!!! important
    put four spaces prior to this text and this will be higlighted in a block :) Just like I did for this sentence
```